dwh/indicator_lib.py

```diff
@@ -140,7 +150,7 @@ def calc_TX_PVLS(patient_agg_obs: DataFrame, VL_code: str,
       lambda a, g: agg_buckets(a, g, end_date),
       T.ArrayType(T.StringType()))
   VL_df = patient_agg_obs.withColumn(
-      'sup_VL', patient_agg_obs[VL_code + '_max_value'] < 150).withColumn(
+      'sup_VL', patient_agg_obs[VL_code + '_max_value'] > failure_threshold).withColumn(
```

Comment:
What is the rationale for changing the direction of this inequality? To `sup_VL` is `True` if viral-load is suppressed, i.e., its max is _lower_ than a threshold, isn't it?

Reply:
the original was correct! I was trying to validate AMPATH's report and forgot to change this. Thanks for flagging this